File: client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/auth/HeaderDto.java
package org.evomaster.client.java.controller.api.dto.auth;

/**
 * A HTTP header, which is a key=value pair
 */
public class HeaderDto {

    /**
     * The header name
     */
    public String name;

    /**
     * The value of the header
     */
    public String value;


    public HeaderDto() {
    }

    public HeaderDto(String name, String value) {
        this.name = name;
        this.value = value;
    }
}

File: outdated/outdated-client-dotnet/src/EvoMaster.Controller.Api/dto/HeaderDto.cs
namespace EvoMaster.Controller.Api {
    public class HeaderDto {
        public HeaderDto() { }

        public HeaderDto(string name, string value) {
            this.Name = name;

            this.Value = value;
        }

        /**
     * The header name
     */
        public string Name { get; set; }

        /**
     * The value of the header
     */
        public string Value { get; set; }
    }
}